recipe/meta.yaml

```diff
@@ -11,10 +11,13 @@ source:
   patches:
     # http://lists.gnu.org/archive/html/bug-ncurses/2011-04/txtkWQqiQvcZe.txt
     - fix.patch
+    - ncurses-5.9-gcc-5.patch
 
 build:
   skip: true  # [win]
-  number: 0
+  number: 5 # [linux64]
+  number: 6 # [linux32]
+  number: 1 # [osx]
```

Comment:
No need to split the build number and jump them to higher number. Just use `number: 1`.